barretenberg/cpp/src/barretenberg/plonk_honk_shared/types/aggregation_object_type.hpp

```diff
@@ -10,7 +10,11 @@ namespace bb {
 constexpr uint32_t PAIRING_POINT_ACCUMULATOR_SIZE = 16;
 // PairingPointAccumulatorIndices represents an array of 16 witness indices pointing to the nested aggregation object.
 using PairingPointAccumulatorIndices = std::array<uint32_t, PAIRING_POINT_ACCUMULATOR_SIZE>;
-// PairingPointAccumPubInputIndices represents an array of 16 public input indices pointing to the witness indices of
-// the nested aggregation object.
-using PairingPointAccumPubInputIndices = std::array<uint32_t, PAIRING_POINT_ACCUMULATOR_SIZE>;
+// PairingPointAccumulatorPubInputIndices represents an array of 16 public input indices pointing to the witness indices
+// of the nested aggregation object.
+using PairingPointAccumulatorPubInputIndices = std::array<uint32_t, PAIRING_POINT_ACCUMULATOR_SIZE>;
+
+constexpr uint32_t IPA_CLAIM_SIZE = 6;
```

Comment:
FYI IIUC `static constexpr` will ensure that this is initialized at compilation time.

Reply:
shouldn't constexpr also initialize it at compilation time?

Reply:
Yes but I mean `static constexpr`